cluster/examples/gitlab/aws/secrets.sh

```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env bash
```

Comment:
Hrm, I guess my editor decided to run https://github.com/mvdan/sh against all the shell scripts under `examples/` too.